tests/python/contrib/test_vitis_ai/infrastructure.py

```diff
@@ -59,7 +59,7 @@ def visit_call(self, call):
 
 def skip_test():
```

Comment:
Overall LGTM! I see that tvm has more or less a standard way of defining require_*() functions in this file and using those functions to skip the tests. Maybe you could define a similar function for vitis_ai.
https://github.com/apache/tvm/blob/main/python/tvm/testing/utils.py